software/webapp/src/main/java/brooklyn/entity/proxy/nginx/NginxSshDriver.java

```diff
@@ -216,6 +220,12 @@ public void install() {
         String withLdOpt = entity.getConfig(NginxController.WITH_LD_OPT);
         if (isMac) withLdOpt = format("-L%s/pcre-dist/lib", getInstallDir()) + (Strings.isBlank(withLdOpt) ? "" : " " + withLdOpt);
         String withCcOpt = entity.getConfig(NginxController.WITH_CC_OPT);
+        
+        if (isMac) {
+            // TODO Upgrade sticky module as soon as a fix for https://bitbucket.org/nginx-goodies/nginx-sticky-module-ng/issue/16/can-not-compile-on-macosx-yosemite
+            // is released and remove this block.
+            withCcOpt = (Strings.isBlank(withCcOpt) ? "" : (withCcOpt + " ")) + "-Wno-error";
```

Comment:
would it make sense to move it into the `withCcOpt` configKey? wouldn't that be beneficial for other distros or just mac?